nkms/keystore/keypairs.py

```diff
@@ -71,7 +72,24 @@ def decrypt(self, message_kit: MessageKit) -> bytes:
 
         :return: bytes
         """
-        return message_kit.decypt(self.privkey)
+        cleartext = pre.decrypt(message_kit.capsule, self._privkey,
```

Comment:
Why not go all the way here and just let `pre.decrypt` just take a `MessageKit` as an arg.  It may even be more intuitive to users who are already familiar with PKE to just pass a private key and a message... `pre.decrypt(message_kit, private_key)`

Reply:
I mean you know I'm :+1: on that.